app/email_branding/rest.py

```diff
@@ -38,24 +39,33 @@ def create_email_branding():
     data = request.get_json()
 
     validate(data, post_create_email_branding_schema)
-
     email_branding = EmailBranding(**data)
+
     if "text" not in data.keys():
         email_branding.text = email_branding.name
 
-    dao_create_email_branding(email_branding)
+    try:
+        dao_create_email_branding(email_branding)
+    except IntegrityError as e:
```

Comment:
Just a note that this constraint means we could have brandings called `Test` and `test` and they wouldn't conflict :/

Reply:
We could potentially solve this by changing the name column to a case insensitive data type: [`citext`](https://www.postgresql.org/docs/current/citext.html). This way we don't need to perform the validation in API and add another query to the process.